source/catalog.html

```diff
@@ -30,189 +31,191 @@
-          </article>
-          </li>
-          <li class="catalog__item">
-            <article class="product__card">
-            <a class="catalog__link-img" href="#!">
-              <img class="catalog__img" width="68" height="78" src="img/catalog-products/catalog-3.png" alt="Зеленая упаковка cat energy 500 грамм со вкусом рыбы">
-            </a>
-            <h3>
-              <a class="catalog__link" href="#!">Cat Energy PRO 500 г</a>
-            </h3>
-            <table class="catalog__details">
-              <tr>
-                <td>Масса</td>
-                <td>500 г</td>
-              </tr>
-              <tr>
-                <td>Вкус</td>
-                <td>Рыба</td>
-              </tr>
-              <tr>
-                <td>Цена</td>
-                <td>700 Р.</td>
-              </tr>
-            </table>
-            <a class="button catalog__button">заказать</a>
-            </article>
-          </li>
-          <li class="catalog__item">
-            <article class="product__card">
-            <a class="catalog__link-img" href="#!">
-              <img class="catalog__img" width="84" height="101" src="img/catalog-products/catalog-4.png" alt="Зеленая упаковка cat energy 1000 грамм со вкусом рыбы">
-            </a>
-            <h3><a class="catalog__link" href="#!">Cat Energy PRO 1000 г</a></h3>
-            <table class="catalog__details">
-              <tr>
-                <td>Масса</td>
-                <td>1000 г</td>
-              </tr>
-              <tr>
-                <td>Вкус</td>
-                <td>Рыба</td>
-              </tr>
-              <tr>
-                <td>Цена</td>
-                <td>1000 Р.</td>
-              </tr>
-            </table>
-            <a class="button catalog__button">заказать</a>
-            </article>
-          </li>
-          <li class="catalog__item">
-            <article class="product__card">
-            <a class="catalog__link-img" href="#!">
-              <img class="catalog__img" width="68" height="78" src="img/catalog-products/catalog-5.png" alt="Синяя упаковка cat energy 500 грамм со вкусом гречки">
-            </a>
-            <h3><a class="catalog__link" href="#!">Cat Energy SLIM 500 г</a></h3>
-            <table class="catalog__details">
-              <tr>
-                <td>Масса</td>
-                <td>500 г</td>
-              </tr>
-              <tr>
-                <td>Вкус</td>
-                <td>Гречка</td>
-              </tr>
-              <tr>
-                <td>Цена</td>
-                <td>400 Р.</td>
-              </tr>
-            </table>
-            <a class="button catalog__button">заказать</a>
+        <div class="container">
+          <h2 class="catalog__title">Каталог продукции</h2>
+          <ul class="catalog__list">
+            <li class="catalog__item container">
+              <article class="product__card">
+              <a class="catalog__link-img" href="#!">
```

Comment:
карточки товаров самодостаточны, их вполне можно сделать бем-блоком. И я уже об этом говорил
```suggestion
              <article class="product-card">
              <a class="product-card__link-img" href="#!">
```